pkgs/development/tools/flatpak-builder/default.nix

```diff
@@ -76,19 +76,17 @@ in stdenv.mkDerivation rec {
       euelfcompress = "${elfutils}/bin/eu-elfcompress";
     })
 
-    # The test scripts in Flatpak repo were updated so we are basing
-    # this on our patch for Flatpak 0.99.
     (substituteAll {
       src = ./fix-test-paths.patch;
       inherit glibcLocales;
-      # FIXME use python3 for tests that rely on python2
-      # inherit python2;
     })
+    ./fix-test-prefix.patch
```

Comment:
This seems upstream-able. Would you be willing to submit a PR upstream and `fetchpatch` it?

Reply:
I tried to find a solution but upstream wasn't really welcoming so I don't think it'll be possible.